src/App.css

```diff
@@ -0,0 +1,232 @@
+.App {
```

Comment:
Please use kebab-case for all class names

Reply:
I forget to delete this file .This file is not referenced by my code.